tests/std/test.lst

```diff
@@ -168,6 +168,7 @@ tests\GH_001017_discrete_distribution_out_of_range
 tests\GH_001059_hyperbolic_truncation
 tests\GH_001086_partial_sort_copy
 tests\GH_001103_countl_zero_correctness
+tests\GH_001105_streambuf

```

Comment:
This is correctly mentioning the newly added test directory in sorted order. However, I recommend making the directory name slightly more detailed. This is important because when a test fails, the name is a reminder of what the test does. Here, `custom_streambuf_throws` would be much more descriptive than just `streambuf`. (In addition to renaming it here, the directory itself should be renamed with `git mv`.)